### PR TITLE
Update deps, Travis -> GitHub Actions, ZEIT -> Vercel.

### PR DESCRIPTION
**Pull request by leerob**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    name: Node.js ${{ matrix.node-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version:
+          - 16
+          - 14
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
```
Comment (styfle) [inline .github/workflows/ci.yaml:16]:
```suggestion
      - uses: actions/setup-node@v2
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
+      - run: npm test
```
Comment (styfle) [inline .github/workflows/ci.yaml:20]:
```suggestion
      - run: yarn install
      - run: yarn test
```

File: license.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-MIT License
+The MIT License (MIT)
 
-Copyright (c) 2017 Zeit, Inc.
+Copyright (c) 2021 Vercel, Inc.
```
Comment (styfle) [inline license.md:3]: I think this should be 2017-2021 but not sure, probably need to ask legal

File: test/test.js [hunk 4]
```diff
@@ -1,29 +1,33 @@
 // Packages
-const test = require('ava')
+const test = require('ava');
```
Comment (styfle) [inline test/test.js:2]: Where are all these formatting changes coming from? Did you forget to add a devDependency such as prettier?

File: license.md (hunk 3 above)
Comment (leerob) [reply license.md:3]: I copied the existing repos (they all use latest version)

File: test/test.js (hunk 4 above)
Comment (leerob) [reply test/test.js:2]: I ran Prettier, yes. We can add that here, or a separate PR?

Comment (leerob) [reply test/test.js:2]: Added here 😄

File: package.json [hunk 5]
```diff
@@ -11,11 +13,16 @@
...
+    "@types/node": "16.6.1",
+    "ava": "3.15.0",
+    "prettier": "2.3.2",
+    "typescript": "4.3.5"
+  },
+  "prettier": {
+    "arrowParens": "always",
+    "singleQuote": true,
+    "tabWidth": 2,
+    "trailingComma": "none"
```
Comment (styfle) [inline package.json:26]:
```suggestion
  "prettier": {
    "singleQuote": true,
    "tabWidth": 2,
    "trailingComma": "all"
```



This should follow the [style-guide](https://github.com/vercel/style-guide)

cc @mrmckeb if we want to make this a public

Comment (leerob) [reply package.json]: A future project that would be good:

1. Updating our top 20 OSS repos to use consistent Prettier / ESLint from style guide
2. Having them all publish ESM

**Pull request merged**